frontend/src/scenes/experiments/experimentLogic.tsx

```diff
@@ -616,6 +622,63 @@ export const experimentLogic = kea<experimentLogicType>([
                 },
             },
         ],
+        requiredEventsBreakdown: [
+            null as RequiredEventsBreakdown | null,
+            {
+                loadRequiredEventsBreakdown: async (): Promise<RequiredEventsBreakdown> => {
+                    const output = {
+                        presentEvents: [] as { event: string; variant: string }[],
+                        missingEvents: [] as { event: string; variant: string }[],
+                    }
+
+                    const { experiment } = values
+                    const { feature_flag, feature_flag_key } = experiment
+                    const events = experiment.filters.events?.map((e) => e.id)
+                    const variants = feature_flag?.filters.multivariate?.variants.map((v) => v.key)
+
+                    if (!events?.length || !variants?.length) {
+                        return output
+                    }
+
+                    const query: HogQLQuery = {
+                        kind: NodeKind.HogQLQuery,
+                        query: `
+                                    SELECT
+                                        event,
+                                        JSONExtractString(properties, '$feature/${feature_flag_key}') AS variant,
+                                        COUNT(*) AS count
+                                    FROM events
```

Comment:
for future reference, you always want a time range filter in these queries, they'll be across all time, and way too slow otherwise.